reference/orders.v2.oas2.yml

```diff
@@ -347,10 +366,63 @@ paths:
                           value: '180'
                         - id: 230
                           value: '192'
+              Pickup Order:
```

Comment:
we may need to update "description" of "post" about the "required fields": either "consignments" or "products" field is required

Reply:
Good catch. Does this address your comment?
https://github.com/bigcommerce/api-specs/pull/315/commits/63628293af4fe7d659707f100c2e34352f309b5d

Reply:
👍